internal/tools/aws/database_multitenant.go

```diff
@@ -0,0 +1,637 @@
+package aws
+
+import (
+	"context"
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/aws/aws-sdk-go/service/rds"
+	gt "github.com/aws/aws-sdk-go/service/resourcegroupstaggingapi"
+	"github.com/aws/aws-sdk-go/service/secretsmanager"
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/mattermost/mattermost-cloud/model"
```

Comment:
I believe that idiomatically we prefer to organize the imports so that Mattermost imports are separate from the 3rd party libraries.